pkg/services/services.go

```diff
@@ -189,13 +187,13 @@ func getKubeletServiceConfiguration(argsFromIginition map[string]string, debug b
 		kubeletServiceCmd = fmt.Sprintf("%s --node-ip=%s", kubeletServiceCmd, NodeIPVar)
 	}
 	return servicescm.Service{
-		Name:                         windows.KubeletServiceName,
-		Command:                      kubeletServiceCmd,
-		Priority:                     1,
-		Bootstrap:                    true,
-		Dependencies:                 []string{windows.ContainerdServiceName},
-		PowershellVariablesInCommand: powershellVars,
-		NodeVariablesInCommand:       nil,
+		Name:                   windows.KubeletServiceName,
+		Command:                kubeletServiceCmd,
+		Priority:               1,
+		Bootstrap:              true,
+		Dependencies:           []string{windows.ContainerdServiceName},
+		PowershellPreScripts:   powershellVars,
```

Comment:
Now this looks a bit odd. Why would `PowershellPreScripts` be referred to as `powershellVars`